apps/patient/src/components/CourierOptions.tsx

```diff
@@ -23,6 +26,7 @@ interface Props {
 }
 
 export const CourierOptions = ({
+  alto,
```

Comment:
can we add the default props for alto and capsule here as opposed to below? bit more legible and streamlined

``` tsx
export const CourierOptions = ({
  alto = false,
  capsule = false, 
 ...
}
```